execution/executor/src/metrics.rs

```diff
@@ -79,6 +79,29 @@ pub static APTOS_EXECUTOR_EXECUTE_BLOCK_SECONDS: Lazy<Histogram> = Lazy::new(||
     .unwrap()
 });
 
+pub static APTOS_EXECUTOR_LEDGER_UPDATE_SECONDS: Lazy<Histogram> = Lazy::new(|| {
+    register_histogram!(
+        // metric name
+        "aptos_executor_ledger_update_seconds",
+        // metric description
+        "The total time spent in ledger update in the block executor.",
+        exponential_buckets(/*start=*/ 1e-3, /*factor=*/ 2.0, /*count=*/ 20).unwrap(),
+    )
+    .unwrap()
+});
+
+pub static APTOS_EXECUTOR_LEDGER_UPDATE_OTHER_SECONDS: Lazy<HistogramVec> = Lazy::new(|| {
```

Comment:
what's wrong with just using `APTOS_EXECUTOR_OTHER_TIMERS_SECONDS`?

Reply:
Nothing wrong, I thought its just better to have a separate counter for breaking down the ledger update phase but change it back if strong opinion about it.